lib/rubocop/cop/rspec/mixin/css_selector.rb

```diff
@@ -0,0 +1,85 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module RSpec
+      # Helps parsing css selector.
+      module CssSelector
```

Comment:
I really like how you’ve grouped common functionality together in this re-usable module.

But I think we should consider not including this module, but instead make the methods “module functions” using `module_function` (or `extend self`, depending on our RuboCop configuration). I say this because the methods are _very_ specialized, and because none of them depend on the cop context, but operate entirely on the single method argument being passed to them.

Reply:
That certainly seems like a better. If I am mistaken in my approach, please point out. I changed it to `module_function`.